blogs/2022/09/15/dev-container-features.md

```diff
@@ -0,0 +1,139 @@
+---
+Order: 76
+TOCTitle: Dev Container Features
+PageTitle: Dev Container Features
+MetaDescription: What are dev container Features and what's new
+Date: 2022-09-15
+Author: Brigit Murtaugh
+---
+
+# Dev Container Features
```

Comment:
I debated about a compound title like that, or just `Dev Container Feature Packs`. I was leaning toward just `Dev Container Features` since features are the real star we're focused on, and "packs" are the mechanism to install them. What do you think?

Reply:
Yeah - only reason I was wondering about it is the question as to whether people would interpret this title as features of Remote - Containers, which is partly why we're having the conversation about adding pack/package/bundle/etc to the end for the tgz.  We could adjust it to say something like "Publishing Dev Container Features" or "Custom Dev Container Features" something like that though which would probably clarify.

Reply:
I like `Custom Dev Container Features` - updated!